src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Active areas of Ethereum research
+description: Explore different areas of open research and learn how to get involved.
+sidebar: true
+lang: en
+---
+
+# Open areas of Ethereum research
+
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline some of the key areas of active research as a rough guide to Ethereum's cutting edge.
+
+## General research resources
+
+Regardless of the specific topic, there is a wealth of information on Ethereum research to be found at [ethresear.ch](https://ethresear.ch) and the [Eth R&D Discord channel](https://discord.gg/n7JxAeRu). These are the primary places where Ethereum researchers discuss the latest ideas and development opportunities.
+
+## Scaling and Performance
+
+There are ongoing efforts to scale Ethereum using a variety of techniques including sharding the blockchain, danksharding and using rollups. Introductory information on scaling Ethereum is available on our [scaling page](/developers/docs/scaling).
+
+### Layer 2
+
+There are now several Layer 2 protocols that scale Ethereum using different techniques for batching transactions and securing them on Ethereum layer 1. This is a very rapidly growing topic with a lot of research and development potential.
+
+#### Links
+
+[Our Layer-2 page](https://ethereum.org/en/layer-2/)
```

Comment:
```suggestion
[Introduction to layer 2](/layer-2/)
```
Reminder to link to relative paths (without language) whenever you link to ethereum.org content: 
https://ethereum.org/en/contributing/style-guide/#internal-links

Reply:
A more general question - what do we think is the right level of depth for these various topics? This is a very introductory resource for beginners IMO. I suspect visitors who are looking into exploring open research would prefer more technical content, e.g.
- https://ethereum.org/en/developers/docs/scaling/optimistic-rollups/
- https://ethereum.org/en/developers/docs/scaling/zk-rollups